tests/unit/test-class-wc-payment-gateway-wcpay.php

```diff
@@ -2069,6 +2115,30 @@ public function test_is_in_test_mode() {
 		$this->assertFalse( $this->wcpay_gateway->is_in_test_mode() );
 	}
 
+	public function test_request_class_example() {
```

Comment:
Let's add a to-do to remove this method before merging.

Reply:
Added to-do remainder.